README.md

```diff
@@ -516,7 +516,7 @@ ls: cannot access '/dev/pmem*': No such file or directory
 
 ## Supported Kubernetes versions
 
-PMEM-CSI driver implements CSI specification version 1.0.0, which only
+PMEM-CSI driver implements CSI specification version 1.0.0, which is only
```

Comment:
While we are at it: "PMEM-CSI implements the CSI specification version 1.x, which is..."

Reply:
This is still open.

Reply:
ah, did not notice all diffs in your line. Changed now.